test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -270,6 +304,12 @@ private string GetConnectionString(string databaseName)
             return $@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename={databaseFile};Database={databaseName};Integrated Security=True;Connect Timeout=120";
         }
 
+        private string GetSQLiteConnectionString(string databaseName)
+        {
+            var databaseFile = SamplesUtils.DatasetUtils.GetFilePathFromDataDirectory($"{databaseName}.sqlite");
```

Comment:
Temporarily the added SQLite database is in the `/test/data` directory. I am starting this review to discuss where this `iris.sqlite` file can be stored, or how it can be automatically generated, similar to how the `iris.mdf` and `iris_log.ldf` are being generated.

Reply:
I think `iris.sqlite` can stay in `/test/data` as it is only around 8 MB, and is easily accessed with a function for accessing this directory. There isn't a need to keep copying this database to the bin working directories for tests, as this database is read-only. What do others think on this?

Reply:
machinelearning-testdata has mostly small datasets and models. Not sure what's a good metric to decide where to store the data.

Reply:
Most other datasets and models on the testdata repo are smaller than this. Also iris.mdf is also around 8mb. So the larger models should go there. And it should live alongside iris.mdf so that the two consistent.

---
In reply to: [418772466](https://github.com/dotnet/machinelearning/pull/5080#discussion_r418772466) [](ancestors = 418772466)

Reply:
Ok, then I will make a separate PR to add `iris.sqlite` into machinelearning-testdata/Microsoft.ML.TestDatabases/TestDatabases/, and make sure that PR gets merged before this PR. Thanks.

Reply:
I've made the PR for adding `iris.sqlite` in machinelearning-testdata. This is the PR: https://github.com/dotnet/machinelearning-testdata/pull/24

It'd be great if someone can approve that PR. :D

Reply:
I have approved this PR and merged the PR, please reduce the code duplication, others look good to me

---
In reply to: [418778046](https://github.com/dotnet/machinelearning/pull/5080#discussion_r418778046) [](ancestors = 418778046)